baselines/dom.generated.d.ts

```diff
@@ -7112,7 +7112,7 @@ interface IDBRequestEventMap {
 }
 
 interface IDBRequest extends EventTarget {
-    readonly error: DOMError;
+    readonly error: DOMException | DOMError;
```

Comment:
Right but that's 100% browser dependent behavior and TS type does not affect it.